src/site/content/en/secure/what-is-mixed-content/index.md

```diff
@@ -113,27 +104,15 @@ Due to the severity of this threat,
 most browsers already block this type of content by default to protect users,
 but functionality varies between browser vendors and versions.
 
-The following contains examples of active mixed content:
-
-<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
-  <iframe
-    src="https://glitch.com/embed/#!/embed/active-mixed-content?previewSize=100"
-    title="active-mixed-content on Glitch"
-    allow="encrypted-media"
-    style="height: 100%; width: 100%; border: 0;">
-  </iframe>
-</div>
-
+This other [Glitch demo](https://active-mixed-content.glitch.me/) contains examples of active mixed content.
 [Load the example over HTTP](http://active-mixed-content.glitch.me/)
 to see the content that's blocked when you [load the example over
-HTTPS](https://active-mixed-content.glitch.me/).
-
-Blocked content will be detailed in the Chrome DevTools Console.
+HTTPS](https://active-mixed-content.glitch.me/). Blocked content will also be detailed in the Issues Tab.
```

Comment:
Suggestion/Nit (not a must for this PR since the original article is like that, but while we're at it): we could add a mention of what other browsers do, in order to make this web.dev article browser-agnostic.
E.g. `Browsers also highlight blocked content in developer tooling: block content issues are detailed in the Issues Tab in Chrome and Edge, and Firefox and Safari log messages in the developer console.`

Reply:
Added below